docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -347,29 +343,25 @@ Release
 
      .. prompt:: bash [vagrant@localhost]$
 
-        cd ../homebrew-tap-${VERSION}
+        cd
+        git clone git@github.com:ClusterHQ/homebrew-tap.git "homebrew-tap-${VERSION}"
+        cd homebrew-tap-${VERSION}
         ../flocker-${VERSION}/admin/make-homebrew-recipe > flocker-${VERSION}.rb
         git add flocker-${VERSION}.rb
         git commit -m "New Homebrew recipe"
         git push
 
-   - Test the new recipe on OS X with `Homebrew`_ installed:
-
-     Try installing the new recipe directly from a GitHub link
-
-     .. prompt:: bash $
-
-        brew install --verbose --debug https://raw.githubusercontent.com/ClusterHQ/homebrew-tap/release/flocker-${VERSION}/flocker-${VERSION}.rb
-        brew test flocker-${VERSION}
+   - Test Homebrew on OS X.
+     ClusterHQ has a Mac Mini available with instructions for launching a Virtual Machine to do this with:
 
-   - Make a pull request:
+     Export the version number of the release being completed as an environment variable:
 
-     Make a `homebrew-tap`_ pull request for the release branch against ``master``, with a ``[FLOC-123]`` summary prefix, referring to the release issue that it resolves.
+     .. prompt:: bash [osx-user]$
 
-     Include the ``brew install`` line from the previous step, so that the reviewer knows how to test the new recipe.
+        export VERSION=0.1.2
 
-   - Do not continue until the pull request is merged.
```

Comment:
Add some sort of warning re "and if tests fail you shouldn't continue, and you may need to edit/replace the thing on master."